src/modules/RA/DataLayer/components/DataLayerForm.js

```diff
@@ -129,7 +129,7 @@ const DataLayerForm = ({
 
       </LazyFormTab>
 
-      <LazyFormTab disabled={external} label="datalayer.form.style" path="style">
+      <LazyFormTab disabled={external} label="datalayer.form.styles.tab">
```

Comment:
Why having remove `path="style"` prop ?

Reply:
I misunderstood the utility of this property.
Fixed